src/SensorReading.ts

```diff
@@ -58,43 +72,44 @@ export class SensorReading {
     }
     return 0;
   }
+
+  static pmToAQandU(pm: number): number {
```

Comment:
if you can find a link that explains this equation and put it in the comment, similar to the aqi calculation, that'd be great.

Reply:
The equation is on PurpleAir's site, when you hover on the `?` next to `Conversion`:

<img width="394" alt="Screen Shot 2020-09-13 at 12 53 33" src="https://user-images.githubusercontent.com/6760/93027185-23b7de80-f5c0-11ea-8d67-4525824f748d.png">